apps/fluent-tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -60,6 +68,17 @@ export const CalloutTest: React.FunctionComponent<{}> = () => {
     setShowCustomizedCallout(false);
   }, [setIsCustomizedCalloutVisible, setShowCustomizedCallout]);
 
+  const onRestoreFocusStandardCallout = React.useCallback(
+    (restoreFocusEvent: OnRestoreFocusEvent) => {
+      if (restoreFocusEvent.nativeEvent.containsFocus && decoyBtn1Ref && decoyBtn1Ref.current) {
```

Comment:
good opportunity for optional chaining :)

Reply:
on containsFocus or decoybtn?  decoybtn has other fun with undefined vs null that makes optional chaining suck (see findNodeHandle PR I just squashed)

Reply:
why not both? I think the whole if/else if block could be something like
```
restoreFocus?.nativeEvent?.containsFocus  ? 
   decoyBtn1Ref?.current?.focus?.()  :
   decoyBtn2Ref?.current?.focus?.();
```
It's probably too hard to read, but the syntax + semantics here should work (undefined and null are both nullish, and that's what matters for optional chaining). Why is optional chaining on decoyBtn bad?